engine/table/src/main/java/io/deephaven/engine/util/PythonScopeJpyImpl.java

```diff
@@ -200,4 +211,12 @@ public static Object convert(PyObject pyObject) {
     public PyDictWrapper globals() {
         return dict;
     }
+
+    @Override
+    public void modifyScope(PyObject pydict) {
+        if (pydict.isNone())
+            moduleDict.set(null);
```

Comment:
Could we just simplify this and `currentScope` by just installing `this.dict` as `moduleDict` when `null` is passed?